spec/src/main/asciidoc/ecosystem.asciidoc

```diff
@@ -92,7 +92,7 @@ If the pod is not live, it will destroy the pod and start a new one. If the appl
 
 
 == The ecosystem
-Service mesh can utilise the endpoint exposed by MicroProfile Health for its liveness probe, so that it can control whether to destroy the pod or not.
+The health check of Istio services can be achieved via Kubernetes liveness and readiness probe, where MicroProfile health attributes via the exposed health check endpoints with some meaningful response. The response of the liveness probe determines whether to recycle the corresponding pod, while the readiness response determines whether requests can be routed.
```

Comment:
My only comment is whether the use of "Istio" here is needed?

Reply:
changed to "services in the mesh" as agreed on today's hangout.